src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class FileWriterServiceImplTest {
+    private static final String OUTPUT_FILE_PATH = "src/test/resources/output-test.csv";
+    private static final String EMPTY_REPORT = "";
+    private static final String INVALID_FILE_PATH = "";
+    private FileWriterServiceImpl fileWriterService;
+
+    @BeforeEach
+    void setUp() {
+        fileWriterService = new FileWriterServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        try {
+            Files.deleteIfExists(Path.of(OUTPUT_FILE_PATH));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @DisplayName("Checking for passing null as report value")
+    @Test
+    void writeReport_nullReport_notOk() {
+        Assertions.assertThrows(RuntimeException.class, () ->
```

Comment:
Please move `Assertions.assertThrows` to the static import. Fix all similar cases for all assertions to simplify the code.
```suggestion
       assertThrows(RuntimeException.class, () ->
```